src/test/pow_tests.cpp

```diff
@@ -16,117 +16,81 @@ BOOST_FIXTURE_TEST_SUITE(pow_tests, BasicTestingSetup)
 /* Test calculation of next difficulty target with no constraints applying */
 BOOST_AUTO_TEST_CASE(get_next_work)
 {
-    const auto chainParams = CreateChainParams(CBaseChainParams::MAIN);
-    int64_t nLastRetargetTime = 1261130161; // Block #30240
-    CBlockIndex pindexLast;
-    pindexLast.nHeight = 32255;
-    pindexLast.nTime = 1262152739;  // Block #32255
-    pindexLast.nBits = 0x1d00ffff;
-    BOOST_CHECK_EQUAL(CalculateNextWorkRequired(&pindexLast, nLastRetargetTime, chainParams->GetConsensus()), 0x1d00d86a);
+    SelectParams(CBaseChainParams::MAIN);
```

Comment:
I understood these tests are rewrite because of the signature of `CalculateNextWorkRequired`. I suggest to keep the original function signature of Bitcoin because Zcash was forked from an early version of Bitcoin. So it will be easier to keep in track of the future updates of Bitcion. However it's up to you to decide.

Reply:
Zcash has BTC core 0.11.2 frozen into their tree